.github/workflows/validate.yml

```
@@ -0,0 +1,55 @@
+name: validate
+
+on:
+  push: {}
+  pull_request: {}
```

I don't think this runs on anything? At least no test run was triggered for this PR, how about?

```suggestion
on:
  push:
    branches:
    - master
  pull_request:
    branches:
    - master
```

Usually in my own projects I only do `pull_request: branches: master` and have branch protection on `master`, not sure if we every push directly to master though 🤔